packages/blade/src/components/Badge/__tests__/Badge.native.test.tsx

```diff
@@ -0,0 +1,134 @@
+import { Badge } from '../Badge';
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.native';
+import { InfoIcon } from '~components/Icons';
+
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
+afterAll(() => jest.restoreAllMocks());
+
+describe('<Badge />', () => {
+  it('should render Badge with default props', () => {
+    const label = 'Label';
+    const { toJSON } = renderWithTheme(<Badge>{label}</Badge>);
+    expect(toJSON()).toMatchSnapshot();
+  });
+  it('should throw an error when no children are passed', () => {
```

Comment:
Let's move the runtime error check at the end. 

and also 🙈 
Add space between each test. 

```diff
+ });
+      
+ it('should render small size Badge', () => {
```